### PR TITLE
Fix: emphasize tentative nature of meeting slot

### PR DESCRIPTION
**Pull request by woutermont**
Action following multiple cases of uncertainty (cf. [matrix](https://matrix.to/#/!rUzJAFxzsTfTWEIQfp:gitter.im/$xEtmuBLbL7tVWIDI8gKVM1wh3u-LLlJIzLsJPOB674g?via=gitter.im&via=matrix.org&via=t2bot.io))

### CODE REVIEW

**Comment by woutermont** [comment]
If this panel also concludes that it will have no more meetings until the decision of the special topic meeting of 2023-10-10,  it might be better to use a text similar to https://github.com/solid/data-interoperability-panel/pull/318/commits/fa0b1b124050c2e20803d9a757315ea4a29dab39.

File: README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ All contributors to any Work Items must be members of the Solid CG. It’s easy
 
 Anyone can join the [Solid WebID Profile chat](https://gitter.im/solid/webid-profile).
 
-Mondays, 15:00 UTC at https://meet.jit.si/solid-profile . Meetings are transcribed and [published](https://github.com/solid/webid-profile/tree/main/meetings/).
+A **tentative** meeting slot is held open on Mondays, 15:00 UTC at https://meet.jit.si/solid-profile. Please refer to the [chat](https://gitter.im/solid/webid-profile) for concrete meeting plans and agendas. Meetings are transcribed and [published](https://github.com/solid/webid-profile/tree/main/meetings/).
```
Comment (VirginiaBalseiro) [inline README.md:82]: We should suggest people can join the Solid CG meetings if they'd like to discuss a WebID-Profile topic.

**Comment by csarven** [comment]
1. Background: https://github.com/solid/specification/pull/458#issue-1390870735 .
2. webid-profile isn't and wasn't intended to be a panel. It is intended as the first task force (before literally calling it as such).
3. webid-profile meetings have been tentative. The team convenes in its Matrix chat and decides whether a meeting will be held or not based on agenda, available participants and so on. I'll update the CG calendar to tentative for the time being.

File: README.md (hunk 1 above)
Comment (woutermont) [reply README.md:82]: Do you mean as a definitive decision to not have WebID Profile specific meetings anymore? Or as an additional suggestion to this note?

Comment (VirginiaBalseiro) [reply README.md:82]: Additionally. As in another avenue to bring up a WebID-Profile topic for discussion, aside from the chat.

File: README.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ All contributors to any Work Items must be members of the Solid CG. It’s easy
 
 Anyone can join the [Solid WebID Profile chat](https://gitter.im/solid/webid-profile).
```
Comment (woutermont) [inline README.md:80]:
```suggestion
Anyone can join the [Solid WebID Profile chat](https://gitter.im/solid/webid-profile), or propose a topic for discussion during the weekly [Solid Community Group meetings](https://github.com/solid/specification/#participation).
```

Comment (woutermont) [reply README.md:80]: @VirginiaBalseiro ☝️ ?

**Pull request merged**